### PR TITLE
Queue list UI

### PR DESCRIPTION
**Pull request by bugsalexander**
"responsive as heck" –stanley liu 2020

### CODE REVIEW

File: packages/app/components/Queue/QueueCard.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+`;
+
+const CenterRow = styled(Row)`
+  align-items: center;
+`;
+
+interface QueueCardProps {
+  rank: number;
+  name: string;
+  questionType: string;
```
Comment (NEUDitao) [inline packages/app/components/Queue/QueueCard.tsx:35]: should this be an enum/type of some type? i know we don't have concrete question types yet, but I think it's better to future proof it now

File: packages/app/components/Queue/QueueList.tsx [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+          <Col span={2}>
+            <HeaderText>wait</HeaderText>
+          </Col>
+          <Col xs={0} lg={2}>
+            <StatusText>status</StatusText>
+          </Col>
+        </CenterRow>
+      </HeaderCard>
+
+      <QueueCard
```
Comment (NEUDitao) [inline packages/app/components/Queue/QueueList.tsx:73]: we could get rid of the dummy data if we just finished the project rq smh my head

**Comment by dajinchu** [comment]
Vercel can deploy frontend now! https://office-hours-git-queue-list.sandboxneu.now.sh/queue

**Comment by willstenzel** [comment]
Screen Shot 2020-05-24 at 10 09 37 AM
Looks solid on mobile. Nice work

File: packages/app/components/Queue/QueueCard.tsx (hunk 1 above)
Comment (liustanley) [reply packages/app/components/Queue/QueueCard.tsx]: yeah def, we made an enum in #5 but i just moved it into /common for this PR

**Pull request merged**